tests/test_client.py

```diff
@@ -542,31 +542,32 @@ def test_replace_global(client: lightkube.Client):
 @respx.mock
 def test_pod_log(client: lightkube.Client):
     result = ['line1\n', 'line2\n', 'line3\n']
+    expected = [_.strip() for _ in result]
```

Comment:
See https://github.com/encode/httpx/pull/2423

Reply:
it's POSSIBLE to instead of this change -- to just ensure the '\n' are present at the end of each line.  
for us at [Charmed-Kubernetes](https://github.com/charmed-kubernetes) this isn't an API i'm often using.  I only included these changes to the log to get the tests to green.  Would you rather a different PR for the changes to the logs?

Reply:
I moved these changes to https://github.com/gtsystem/lightkube/pull/46